From ab9015ccde9ed33001150810b63fb032666c8a21 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 16:55:05 +0300
Subject: [PATCH] Makefile: release: ignore delete errors

If adding a new repository with non existing release pointer,
we end up with failure.
Ignore delete errors.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index f175390..d71277f 100644
--- a/Makefile
+++ b/Makefile
@@ -72,7 +72,7 @@
 		git tag -afm "Release Habanalabs $(HL_FULL_VER)" $(HL_FULL_VER)
 	@repo forall $(RELEASE_REPOS) -c \
 		git tag -f $(RELEASE_TAG_POINTER)
-	@repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
+	@-repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
 	@wget http://192.168.0.4:8080/job/goya%20release%20test/build?token=goya-release
